app/views/broker_agencies/profiles/_messages.html.erb

```diff
@@ -1,7 +1,7 @@
 <div class="container">
-  <h3> Broker Messages </h3>
+  <h3><%= l10n("broker_messages")%></h3>
   <%= render 'inbox' %>
 
-  <%= link_to "Send Secure Message to #{site_short_name}".html_safe, broker_agencies_inbox_msg_to_portal_path(inbox_id: @provider.id,
+  <%= link_to l10n('broker_agencies.profiles.send_secure_message', site_short_name: site_short_name).html_safe, broker_agencies_inbox_msg_to_portal_path(inbox_id: @provider.id,
```

Comment:
You can pull all the formatting into the l10n() method by changing `l10n('broker_agencies.profiles.send_secure_message', site_short_name: site_short_name).html_safe` to `l10n('broker_agencies.profiles.send_secure_message_html', site_short_name: site_short_name)`